lib/presentation/component/workout_exercise_card.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
Let's move this file to the `edit_workout` package since it relies on the `edit_workout_bloc`. We can update it later if we decide to make this a more generic component